Shared/Samples/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotationView.Model.swift

```diff
@@ -0,0 +1,167 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Foundation
+
+extension EditFeaturesWithFeatureLinkedAnnotationView {
+    /// The view model for the sample.
+    @MainActor
+    final class Model: ObservableObject {
+        /// The feature currently selected by the user.
+        @Published private(set) var selectedFeature: Feature?
+        
+        /// A map with a light gray basemap.
+        let map: Map = {
+            let map = Map(basemapStyle: .arcGISLightGray)
+            
+            // Initially centers the map in Loudoun Country, VA, USA.
+            map.initialViewpoint = Viewpoint(latitude: 39.0204, longitude: -77.4159, scale: 2256)
+            return map
+        }()
+        
+        /// A URL to the temporary file containing the geodatabase.
+        private let temporaryGeodatabaseURL = FileManager
+            .createTemporaryDirectory()
+            .appending(component: "LoudounAnno.geodatabase")
+        
+        /// The building number and street name of the selected feature.
+        var selectedFeatureAddress: (buildingNumber: Int32?, streetName: String?) {
+            let buildingNumber = selectedFeature?.attributes[.addressFieldKey] as? Int32
+            let streetName = selectedFeature?.attributes[.streetNameFieldKey] as? String
+            return (buildingNumber, streetName)
+        }
```

Comment:
```suggestion
        var selectedFeatureAddress: (buildingNumber: Int32, streetName: String)? {
            if let buildingNumber = selectedFeature?.attributes[.addressFieldKey] as? Int32,
               let streetName = selectedFeature?.attributes[.streetNameFieldKey] as? String {
                return (buildingNumber, streetName)
            } else {
                return nil
            }
        }
```
Since the number-name pair is used together, instead of making them optional separately, make the whole tuple optional when either one of them is `nil`.